catalog-spec/catalog-spec.md

```diff
@@ -1,7 +1,8 @@
-# STAC Catalog Specification
+# STAC Catalog Specification <!-- omit in toc --> 
```

Comment:
What is that?

Reply:
Which ones do you use? Sounds useful.

Reply:
It's Visual Studio Code "Markdown All in One" plugin. I have on my list to add it to CONTRIBUTING.md 

I had to add a TOC to Item, so I added `- [Overview](#overview)`, and then the rest of it was populated when I saved by the plugin.